modules/Project/views/view.ganttchart.php

```diff
@@ -25,14 +25,6 @@
 
 class ProjectViewGanttChart extends ViewDetail
 {
-
-    //Constructor
-    public function __construct()
```

Comment:
Not big deal removing this constructor as it was calling `SugarView` the grand-parent constructor, it will not bypass the parent construction anyways. It should simple call `parent::__construct();`. The only issue, it does not follow the standard of what it has been done in the rest of the patch.